oarepo_requests/ui/theme/assets/semantic-ui/js/oarepo_requests_ui/record-requests/components/RequestList.jsx

```diff
@@ -36,16 +36,17 @@ export const RequestList = ({ requests, requestTypes, requestModalType }) => {
           <RequestModal key={request.id} request={request} requestTypes={requestTypes} requestModalType={modalType}
             triggerButton={
               <List.Item as="a" key={request.id}>
-                <List.Content floated="right" verticalAlign="middle">
+                <List.Content floated="right" verticalAlign="middle" style={{ fontSize: "0.8rem" }}>
```

Comment:
Avoid inlined style, noone can reuse & customize that (and we are in public library here)